CMakeLists.txt

```diff
@@ -1,6 +1,12 @@
 project(Zephyr-Kernel VERSION ${PROJECT_VERSION})
 enable_language(C CXX ASM)
 
+# *DOCUMENTATION*
+# To see a list of typical targets execute "make usage"
```

Comment:
should this be "cmake usage"?

Reply:
Unfortunately not. CMake does not support having a CLI that looks like that.

The user must first generate project files and then invoke the generated project to get a help text.

Reply:
The closest you could get is something like this;

cmake -Dusage=1

which does not look good.